orchagent/aclorch.cpp

```diff
@@ -4660,16 +5004,43 @@ void AclOrch::doAclRuleTask(Consumer &consumer)
             }
 
             if (bHasIPV4 && bHasIPV6)
-	    {
-		    if (type == TABLE_TYPE_L3V4V6)
-		    {
-			    SWSS_LOG_ERROR("Rule '%s' is invalid since it has both v4 and v6 matchfields.", rule_id.c_str());
-			    bAllAttributesOk = false;
-		    }
-	    }
+	        {
```

Comment:
Please fix the alignment.

Reply:
i think this is github issue. i dont see the alignment problem in other editors.